src/promptflow/promptflow/_cli/_pf/_tool.py

```diff
@@ -65,9 +68,38 @@ def add_parser_init_tool(subparsers):
     )
 
 
+def add_parser_list_tool(subparsers):
+    """Add tool list parser to the pf tool subparsers."""
+    epilog = """
+Examples:
+
+# List all package tool in the environment:
```

Comment:
Is this a public API? may update existing doc for CLI in next PR. https://microsoft.github.io/promptflow/reference/pf-command-reference.html#pf-run-create

Reply:
Update the cli doc